### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/server-platforms/symfony/01-login.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ This tutorial and seed project have been tested with the following:
 * Symfony 2.8
 :::
 
+<%= include('../_includes/_package', {
```
Comment (mpaktiti) [inline articles/server-platforms/symfony/01-login.md]: @mehreencs87 This `../_includes/_package` does not exist, the valid one is `../../_includes/_package`. Please update this.

Also, you should add one more line here, according to the latest [contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects), the `githubUrl` option. Its value should be the full URL to the sample, i.e. `https://github.com/auth0-samples/auth0-symfony-php-web-app/tree/master/00-Starter-Seed`

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by mehreencs87** [comment]
Check the latest commit for update. @mpaktiti

File: articles/server-platforms/symfony/01-login.md [hunk 2]
```diff
@@ -11,6 +11,15 @@ This tutorial and seed project have been tested with the following:
 * Symfony 2.8
 :::
 
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-symfony-php-web-app/tree/master/00-Starter-Seed',
+  pkgRepo: 'auth0-symfony-php-web-app',
+  pkgBranch: 'master',
+  pkgPath: '00-Starter-Seed',
+  pkgFilePath: null,
+  pkgType: 'null'
```
Comment (mpaktiti) [inline articles/server-platforms/symfony/01-login.md]: @mehreencs87 The zip download gives an error `{"status":400,"message":"Invalid type.","error":{"status":400}}` The valid values are: server, js, replace, **none** (see: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects)

The download URL is you want to check yourself is: https://auth0.com/docs/package/auth0-symfony-php-web-app/master?path=00-Starter-Seed&file_path=&type=null

**Comment by mehreencs87** [comment]
check the latest commit.

File: articles/server-platforms/symfony/01-login.md [hunk 3]
```diff
@@ -11,6 +11,15 @@ This tutorial and seed project have been tested with the following:
 * Symfony 2.8
 :::
 
+<%= include('../../_includes/_package', {
```
Comment (mpaktiti) [inline articles/server-platforms/symfony/01-login.md:14]: @mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
  Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti updated, check the latest commit.

**Pull request merged**